### PR TITLE
New hat: antlers!

### PR DESCRIPTION
**Pull request by jan-antilles**
[FEATURE][SPRITES]
## About the PR 
This adds a new hat to the game: antlers! It is being added just in time for Spacemas for festive reindeer antlers, but it can stay in all year round as there is nothing specifically Christmassy about it. It is added to the general hat pool, as well as to the Clothing Booth.

## Why's this needed? 
It's cute and festive.

## Changelog

```
(u)Jan.antilles:
(+)Added antlers as a new hat, for Spacemas and beyond! The hat is available in the Clothing Booth.
```

### CODE REVIEW

**Review by flrsh — APPROVED** [review]
These are super cute!

File: code/modules/vending/clothingbooth_items.dm [hunk 1]
```diff
@@ -616,3 +616,9 @@
 	name = "Green Visor"
 	category = "Accessories"
 	cost = PAY_TRADESMAN/5
+
+/datum/clothingbooth_item/antlers
+	path = /obj/item/clothing/head/antlers
+	amount = 3
+	cost = 200
```
Comment (mordent-goonstation) [inline code/modules/vending/clothingbooth_items.dm:623]: Can get rid of this line.

File: code/obj/item/clothing/hats.dm [hunk 2]
```diff
@@ -1212,3 +1212,13 @@
 	desc = "The Chief of Cleaning, the Superintendent of Scrubbing, whatever you call yourself, you know how to make those tiles shine. Good job."
 	icon_state = "janitorberet"
 	item_state = "janitorberet"
+
+/obj/item/clothing/head/antlers
+	name = "antlers"
+	desc = "Be a deer and wear these, won't you?"
+	icon = 'icons/obj/clothing/item_ears.dmi'
```
Comment (mordent-goonstation) [inline code/obj/item/clothing/hats.dm:1219]: ~Should probably put it in `item_hats.dmi` (which would mean you don't need this line as that's the default for `/obj/item/clothing/head`?~

EDIT: Oh, I see that you were probably copying cat ears? Okay, that's fine.

File: code/obj/item/clothing/hats.dm [hunk 3]
```diff
@@ -1212,3 +1212,13 @@
 	desc = "The Chief of Cleaning, the Superintendent of Scrubbing, whatever you call yourself, you know how to make those tiles shine. Good job."
 	icon_state = "janitorberet"
 	item_state = "janitorberet"
+
+/obj/item/clothing/head/antlers
+	name = "antlers"
+	desc = "Be a deer and wear these, won't you?"
```
Comment (mordent-goonstation) [inline code/obj/item/clothing/hats.dm:1218]: 👍 Puns good.

**Review by mordent-goonstation — CHANGES_REQUESTED** [review]

File: code/modules/vending/clothingbooth_items.dm (hunk 1 above)
Comment (jan-antilles) [reply code/modules/vending/clothingbooth_items.dm]: Aha, thanks for noticing that! I started working on this before merging in all of the economy changes. My bad.

**Review by mordent-goonstation — APPROVED** [review]

**Pull request merged**